src/index.html

```diff
@@ -7,8 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img src="./images/logo.png" alt="moyo-1">
+      </a>
+      <nav class="nav">
+        <ul>
+          <li>
+            <a class="nav__link is-active" href="index.html">Apple</a>
+          </li>
+          <li>
```

Comment:
Add a blank line between the links
```suggestion
          </li>
          
          <li>
```